source/common/network/io_socket_handle_impl.cc

```diff
@@ -120,8 +131,14 @@ Api::IoCallUint64Result IoSocketHandleImpl::writev(const Buffer::RawSlice* slice
   if (num_slices_to_write == 0) {
     return Api::ioCallUint64ResultNoError();
   }
-  return sysCallResultToIoCallResult(
+  auto result = sysCallResultToIoCallResult(
       Api::OsSysCallsSingleton::get().writev(fd_, iov.begin(), num_slices_to_write));
+
+  // Some tests try to read without initializing the file_event.
```

Comment:
// Some tests try to **write** without

Reply:
fixed in b248f16d29e74be7a09518fc4c1941fda208571a